mix.exs

```diff
@@ -38,7 +38,7 @@ defmodule Membrane.TelemetryMetrics.Mixfile do
   defp deps do
     [
       {:telemetry, "~> 1.0"},
-      {:telemetry_metrics, "~> 0.6.1"},
+      {:telemetry_metrics, "~> 1.0"},
```

Comment:
```suggestion
      {:telemetry_metrics, "~> 1.0 or ~> 0.6.1"},
```
Please set `@version` to `0.1.1`, so we will be able to release your change at once.

Thank you very much for your contribution!